bluemira/geometry/shell.py

```diff
@@ -110,7 +110,7 @@ def shells(self) -> tuple:
         """
         The shells of the shell. By definition a list of itself.
         """
-        return tuple(self)
+        return tuple([self])
```

Comment:
Does this not also need to be done on edges wires and faces functions?

Reply:
Not sure what you mean, for the others the inputs to the tuple are already iterables

Reply:
Oh sorry, talking rubbish read the diff the wrong way round